src/containers/community/CommunityPosts.js

```diff
@@ -68,25 +83,64 @@ export class CommunityPosts extends Component {
     let { location: { query }, dispatch, community, params: { id } } = this.props
     const { currentUser } = this.context
 
+    let module
+
+    if (groupUser(currentUser.id, 'In-Feed Engagement Modules') === 0) {
+      const showPopularSkills = community.memberCount >= MIN_MEMBERS_FOR_SKILLS_MODULE
+      const showPostPrompt = community.postCount >= MIN_POSTS_FOR_POST_PROMPT_MODULE
+
+      const { moduleChoice } = this.state
+
+      module = {
+        id: -1,
+        type: 'module'
+      }
+
+      if (showPopularSkills) {
```

Comment:
i think this would read a bit better if you did 
```js
if (showPopularSkills && showPostPrompt)
else if (showPopularSkills)
else if (showPostPrompt)
else
```